net-fs/samba/files/samba-4.11-fix-glibc-2.32-function-collisions.patch

```diff
@@ -0,0 +1,76 @@
+Quick-and-dirty fix for Samba build issue with glibc-2.32,
```

Comment:
Quick-and-dirty conveys zero information. I'd rather this explained the actual issue. Something like "Rename Samba functions that collide with new functions shipped in glibc-2.32" or similar.